### ISSUE TITLE
EXCLUDES not working as expected

### ISSUE THREAD

**Issue by PaBLoX-CL** [thoughtbot/rcm#11]
I have the following files in my `~/.dotfiles` directory:

```
config/systemd/user
config/xfce4
systemd
```

I only want to exclude `config/systemd/*`,  so I've tried the following:

```
EXCLUDES="config/systemd" 
```

``` shell
$ lsrc -v 
/home/pablo/.config/systemd/user/default.target:/home/pablo/.dotfiles/config/systemd/user/default.target
/home/pablo/.config/systemd/user/dropbox@.service:/home/pablo/.dotfiles/config/systemd/user/dropbox@.service
/home/pablo/.config/systemd/user/empty.target:/home/pablo/.dotfiles/config/systemd/user/empty.target
/home/pablo/.config/systemd/user/gnome-do@.service:/home/pablo/.dotfiles/config/systemd/user/gnome-do@.service
/home/pablo/.config/systemd/user/numlockx@.service:/home/pablo/.dotfiles/config/systemd/user/numlockx@.service
[ ... a lot of output removed .... ]
/home/pablo/.systemd:/home/pablo/.dotfiles/systemd
```

```
EXCLUDES="systemd" # or "*:systemd"
```

``` shell
$ lsrc -v 
/home/pablo/.config/xfce4/......
[ ... a lot of output removed .... ]
```

Following the instructions for the lsrc man page, I followed the idea of the thoughbot-dotfiles example:

```
EXCLUDES="config:systemd" # or "config:*systemd*"
```

``` shell
$ lsrc -v 
/home/pablo/.config/systemd/user/default.target:/home/pablo/.dotfiles/config/systemd/user/default.target
/home/pablo/.config/systemd/user/dropbox@.service:/home/pablo/.dotfiles/config/systemd/user/dropbox@.service
/home/pablo/.config/systemd/user/empty.target:/home/pablo/.dotfiles/config/systemd/user/empty.target
/home/pablo/.config/systemd/user/gnome-do@.service:/home/pablo/.dotfiles/config/systemd/user/gnome-do@.service
/home/pablo/.config/systemd/user/numlockx@.service:/home/pablo/.dotfiles/config/systemd/user/numlockx@.service
[ ... a lot of output removed .... ]
/home/pablo/.systemd:/home/pablo/.dotfiles/systemd
```

Same thing... am I misunderstanding how it works?

**Comment by mike-burns** [comment]
Captured in a test on the `exclude-globs` branch: https://github.com/thoughtbot/rcm/blob/exclude-globs/test/lsrc-excludes.t#L22-L32

**Comment by mike-burns** [comment]
This might be fixed by #192.
